client/controllers/incidentReports/incidentTable.coffee

```diff
@@ -67,6 +67,16 @@ Template.incidentTable.events
     changeIncidentStatus(false, instance)
     event.currentTarget.blur()
 
+  'click .add': (event, instance) ->
+    instance.data.incidents.find({selected: true}).forEach (incident) ->
+      incident = _id: incident._id
+      incident.userEventId = Template.instance().data.sourceId.get()._str
```

Comment:
I think we want to offer a list of existing events to choose from, then set the `userEventId` to the ID of the chosen event. This will link the source associated with the IR as the event.